nni/retiarii/converter/graph_gen.py

```diff
@@ -100,46 +87,60 @@ def create_prim_constant_node(ir_graph, node, module_name):
 
 def handle_prim_attr_node(node):
     assert node.hasAttribute('name')
-    assert node.inputsAt(0).debugName() == 'self'
-    assert node.inputsAt(0).unique() == 0
     attrs = {'name': node.s('name'), 'input': node.inputsAt(0).debugName()}
     return node.kind(), attrs
 
 def _remove_mangle(module_type_str):
     return re.sub('\\.___torch_mangle_\\d+', '', module_type_str)
 
-def remove_unconnected_nodes(ir_graph):
+def remove_unconnected_nodes(ir_graph, targeted_type):
     """
     Parameters
     ----------
     ir_graph : Graph
         our ir graph representation
+    targeted_type : str
+        nodes with ```targeted_type``` will be removed from graph is their fanout is 0
```

Comment:
is -> if?

Suggest make ``targeted_type`` optional, or make the function name specific, such as ``remove_unconnected_nodes_with_type``.

Reply:
updated